sovryn-token-bridge/bridge/contracts/AllowTokens.sol

```diff
@@ -163,5 +172,16 @@ contract AllowTokens is IAllowTokens, Ownable {
         minAllowedToken[token] = _minAmount;
         emit FeeAndMinPerTokenChanged(token, _feeConst, _minAmount);
     }
+    function setMaxPerToken(address token, uint256 _maxAmount) external onlyOwner {
+        require(_maxAmount <= maxTokensAllowed, "AllowTokens: Max Tokens should be equal or smaller than Max Tokens");
+        require(minAllowedToken[token] <= _maxAmount, "AllowTokens: maxAllowedToken[] should be equal or bigger than minAllowedToken[]");
```

Comment:
You could also validate this in `setFeeAndMinPerToken` (if `maxAllowdToken[token] != 0`)